apps/system/test/unit/multi_screen_controller_test.js

```diff
@@ -39,12 +52,14 @@ suite('system/MultiScreenController', function() {
   }
 
   setup(function() {
+    realApplications = window.applications;
+    window.applications = MockApplications;
```

Comment:
Since you've added "Applications" in `MocksHelper`, you shouldn't need to replace it by yourself.